spec/input_checkers_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'input_checkers'
+require 'BCrypt'
+require 'pg'
+
+include BCrypt
+
+describe InputCheckers do
+
+  before(:each) do
+    connection = PG.connect(dbname: 'chitter_test')
+    password = Password.create("testpassword")
+    connection.exec("INSERT INTO users (name, user_name, email, password) VALUES('test name', 'test user name', 'test@email.com', $txt$#{password}$txt$);")
```

Comment:
You're creating this user before each test... I don't see anywhere where you're deleting it. Or is that part of a helper somewhere?